src/App.css

```diff
@@ -170,15 +170,15 @@
   left: 0;
   right: 0;
   bottom: 0;
-  /* background: rgba(0, 0, 0, 0.7); */
+  background: rgba(0, 0, 0, 0.5);
   display: flex;
   justify-content: center;
   align-items: center;
 }
 .modal-box {
   position: relative;
   background: white;
-  width: 80%;
+  width: 309px;
```

Comment:
Any reason why you changed this to a fixed px vs a %?

Reply:
I did this because we were going to make this used on phone for now. So, I thought people can later on change the px using media query.
I will change them back to percentage.